polkadot/node/network/bridge/src/tx/tests.rs

```diff
@@ -334,7 +368,9 @@ fn send_messages_to_peers() {
 	});
 }
 
+// TODO: discuss with parachains engineering
```

Comment:
@sandreim @vstakhov @alindima 

I mentioned this in Element but because `NetworkBridge` is depending on `[cfg(feature = "network-protocol-staging")]` to specify the used protocol version, this and the other test in `NetworkBridgeTx` won't work anymore without enabling this feature and I couldn't get this test to run with any of the `cargo test --features` combinations I tried so I was wondering do you have any thoughts on what to do this test?